public/app/features/explore/RichHistory/RichHistoryStarredTab.tsx

```diff
@@ -111,7 +111,7 @@ export function RichHistoryStarredTab(props: Props) {
       <div className={styles.containerContent}>
         <div className={styles.selectors}>
           {!activeDatasourceOnly && (
-            <div className={styles.multiselect}>
+            <div className={styles.multiselect} aria-label="filter datasources">
```

Comment:
Nitpick: majority of our aria-labels start with a capital letter. Shouldn't we do the same there?